README

```diff
@@ -1 +1,6 @@
 This is an attempt to build Erlang NIF around gpgme
+
+
+## License 
```

Comment:
thanks for explicitly mentioning it.

Reply:
No worries - I'll have another go at trying to get the common test working once I have less work on - my original reason for trying to bring it up to rebar3 has gone now - but I still think it's a cool library - everyone's using keybase now but there's plenty gpg is still good for.